.circleci/config.yml

```diff
@@ -455,11 +455,11 @@ workflows:
       - build-and-lint-debug
 
       # Build using macOS
-      - build-macos:
-          name: build-macos-xcode-<< matrix.xcode-version >>
-          matrix:
-            parameters:
-              xcode-version: ["11.7.0", *default-xcode-version]
+      # - build-macos:
```

Comment:
Hmm... I think the right approach is to fix the build if we need to support this target...